components/dropdownbutton/icons.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Icons
+page_title: DropDownButton Icons
```

Comment:
This seems like a pattern used by other feature-related articles.
```suggestion
page_title: Blazor DropDownButton - Icons
```

Reply:
That's correct, there is such a patter. However, we do not include the Blazor keyword in the page title. I will update the titles of all articles.